examples/00-load/create-poly.py

```diff
@@ -16,15 +15,66 @@
 ###############################################################################
 # A PolyData object can be created quickly from numpy arrays.  The vertex array
 # contains the locations of the points in the mesh and the face array contains
-# the number of points of each face and the indices of the vertices which comprise that face.
+# the number of points of each face and the indices of the vertices which
+# comprise that face.
 
 # mesh points
 vertices = np.array([[0, 0, 0], [1, 0, 0], [1, 1, 0], [0, 1, 0], [0.5, 0.5, -1]])
 
 # mesh faces
-faces = np.hstack([[4, 0, 1, 2, 3], [3, 0, 1, 4], [3, 1, 2, 4]])  # [square, triangle, triangle]
+faces = np.hstack(
+    [
+        [4, 0, 1, 2, 3],  # square
+        [3, 0, 1, 4],  # triangle
+        [3, 1, 2, 4],  # triangle
+    ]
+)
 
 surf = pv.PolyData(vertices, faces)
 
 # plot each face with a different color
-surf.plot(scalars=np.arange(3), cpos=[-1, 1, 0.5])
+surf.plot(
+    scalars=np.arange(3),
+    cpos=[-1, 1, 0.5],
+    show_scalar_bar=False,
+    show_edges=True,
+    line_width=5,
+)
+
+
+###############################################################################
+# Polyhedral PolyData
+# ~~~~~~~~~~~~~~~~~~~
+# Create a three face polyhedral mesh directly from points and faces.
```

Comment:
Could we rename this? "Polyhedral" sounds like it has to do with polyhedra, but a polyhedron is typically a closed surface as far as I know. This is just a collection of polygons. But if there's some VTK use of this term then I can accept it.

Reply:
```suggestion
# Polygonal PolyData
# ~~~~~~~~~~~~~~~~~~
# Create a three face polygonal mesh directly from points and faces.
```